pydantic_core/_pydantic_core.pyi

```diff
@@ -1,8 +1,6 @@
 import decimal
 import sys
-from typing import Any, Callable, Generic
-
-from typing_extensions import TypeVar
```

Comment:
is `TypeVar` available in `typing` for all python versions supported? If not, I guess there might be a good reason to use `typing_extensions` as we do below?

Reply:
This originated in #643, I saw no reason to use `typing_extensions` here, `TypeVar` is part of the original PEP 484 added in Python 3.5.